.cirun.yml

```diff
@@ -0,0 +1,8 @@
+runners:
+  - name: long-runner
+    cloud: aws
+    instance_type: c5d.4xlarge
+    machine_image: ami-0048c368c80e3ee99
+    preemptible: false
+    workflow: .github/workflows/conda-build.yml
+    count: 2
```

Comment:
@aktech, I tried a few different instance_types and they all fail with `no space left on device` while pulling the docker image. Any ideas?

Reply:
Yes, that's expected with docker images. The default size of instance is less than 10 GB, which is not really enough.  

I can potentially add the setting/configuration in the `.cirun.yml` to increase disk size but that depends on disk type etc and there are lot of permutations for the same.

So, I would suggest to create a custom AMI (say on top of Ubuntu 20.04) with disk size something like say 30 GB or more and use that AMI ID.